package.json

```diff
@@ -6,7 +6,8 @@
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1",
         "start": "webpack-dev-server --mode development --open --hot --port 3000",
-        "build": "mkdir -p dist && webpack --mode production"
```

Comment:
Does dropping the `-p` cause problems on multiple builds?

Reply:
Wasn't win10 compatible. Didn't mean to commit. Will revert

Reply:
I should setup some kind of containerised dev env 🤔